homeworks/17/chat-system/src/main/java/org/example/Client.java

```diff
@@ -0,0 +1,170 @@
+package org.example;
+
+import java.io.*;
+import java.net.Socket;
+import java.security.InvalidParameterException;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Client {
+    private Socket socket;
+    private BufferedReader bufferedReader;
+    private BufferedWriter bufferedWriter;
+    private String username = "Anonimen";
```

Comment:
There should not be a name for a new user. The should name themselves after connection.